src/fontman/ttf_font.cpp

```diff
@@ -79,17 +79,24 @@ bool TtfFont::loadFont(const std::string &path)
 {
     SDL_assert_release(g_ft);
     FT_Error error = FT_New_Face(g_ft, path.c_str(), 0, &m_face);
-    SDL_assert(error == 0);
     if(error)
+    {
+        pLogWarning("Failed to load the font: %s", FT_Error_String(error));
```

Comment:
Nice, I was hoping we could do something like this but didn’t know whether FreeType had error strings.

Reply:
I also found that it's need to enable them at build, otherwise they doesn't work by default.